designer/client/designer.tsx

```diff
@@ -99,7 +99,7 @@ export default class Designer extends Component<Props, State> {
       <FeatureFlagProvider>
         <DataContext.Provider value={dataContextProviderValue}>
           <FlyoutContext.Provider value={flyoutContextProviderValue}>
-            <div id="app">
+            <div id="designer">
```

Comment:
`#app` has styles associated with it in `index.scss`, does dropping them have any impact?

Reply:
Doesn't seem to have any inpact. The issue was there was two divs similar to 
`<div id="app">
    <div id="app">`
We could duplicate the style for designer?